<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatRoomWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutInformation">
     <item>
      <widget class="QTextBrowser" name="txtMessages">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listUsers">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutInput">
     <item>
      <widget class="QLineEdit" name="txtInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>txtMessages</sender>
   <signal>anchorClicked(QUrl)</signal>
   <receiver>ChatWidget</receiver>
   <slot>anchorClicked(QUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>anchorClicked(QUrl)</slot>
 </slots>
</ui>
